### PR TITLE
Update quill to version 1.5.2

### PR DESCRIPTION
**Pull request by odygrd**
Specify library name and version:  **quill/1.5.2**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by prince-chrismc** [comment]
🙈  And older versions of quil are using internal APIs of fmt... 

Unfortunately, CCI does not support removing versions since it will break consumers, if you can please revert that change it would be appreciated...

To workaround this...

You'll have to do something like this https://github.com/conan-io/conan-center-index/blob/master/recipes/spdlog/all/conanfile.py#L51 to have different versions of fmt...

And if you could sprinkle something like this https://github.com/conan-io/conan-center-index/blob/master/recipes/spdlog/all/conanfile.py#L88 that would be even better but optional

**Comment by odygrd** [comment]
Thanks @prince-chrismc I have reverted that change and also implemented something similar as you suggested

**Review by prince-chrismc — APPROVED** [review]
LGTM! So many people will be happy to have a newer version of fmt 🥇

**Comment by prince-chrismc** [comment]
> Would it be possible to restart the CI please ? It looks stuck

Sadly I do not have the permissions since I am a community member, not a part of the CCI team.

As a personal comment, if you can refrain from squashing it makes it easier to see the changes when reviewing, and it's greatly appreciated!

File: recipes/quill/all/conanfile.py
Comment (uilianries) [inline recipes/quill/all/conanfile.py:88]: Good catch!

**Review by uilianries — APPROVED** [review]
LGTM

**Comment by uilianries** [comment]
>  Would it be possible to restart the CI please ? It looks stuck

CCI starts a new build for every new push. Internal errors should be re triggered by admins (they receive a notification about the problem). If you really believe it should be restarted manually because a delay, you can close, wait 10 secs and open the PR again, it should trigger CCI build again.

**Comment by madebr** [comment]
Hey! Stay out of my builds!
See https://github.com/conan-io/conan-center-index/pull/1510#issuecomment-724746378
😄😆🤣

**Review by madebr — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]

**Comment by prince-chrismc** [comment]
@odygrd There are merge conflicts you will need to solve which are preventing the merge

**Comment by odygrd** [comment]
fixed merge conflicts - thanks for reviewing

**Review by SSE4 — APPROVED** [review]

**Comment by prince-chrismc** [comment]
Seems the CCI bot is not able to merge this PR automatically... UI shows the right number of approvals 😕

**Comment by madebr** [comment]
@danimtb @uilianries 
Please come rescue us! :horse_racing:

**Comment by danimtb** [comment]
Unfortunately, the automatic merge is not able to merge this PR.

The issue here is that reviews are associated with a commit in the PR, and we have just discovered that if you push new commits that do not affect the files changes in the PR (like a merge from master or an empty commit), the reviews are not dismissed even if that feature (dismiss stale reviews) is enabled for the branch.
I have opened an issue to try to solve it but at first glance, it could be more complicated than expected.

So, for this PR the automatic merge reports that the only one with an approval in the latest commit is SSE4 🤦‍♂️

**Review by madebr — APPROVED** [review]

**Review by danimtb — APPROVED** [review]

**Comment by jgsogo** [comment]
Hi! This PR broke the line-endings of two files in the `quill` recipe. It didn't take into account gitattributres from https://github.com/conan-io/conan-center-index/pull/3468 because it was generated/compiled before merging them to `master` 😓 

I've prepared #3599 to fix it.

**Pull request merged**
